app_server/public/js/modules/mockdata/user.json

```diff
@@ -0,0 +1 @@
+{ "userId": 2, "username": "alice", "platform_type": "facebook", "alias": "queenofhearts", "description": "I'm Mab, queen of ice and darkness", "email": "alice42@wonderland.org", "createdAt": "2011-04-10T20:09:31Z", "updatedAt": "2014-03-03T18:58:10Z" }
```

Comment:
I'll probably keep `json` files without the new line?
I'm fine either way though, should I keep the convention for them with or without?